### DISCUSSION TITLE
There is a burning pain in my soul

### DISCUSSION THREAD

**Discussion by PatheticMustan** [Prodigy-Hacking/ProdigyMathGameHacking#532] [(D) Developer]
![71C4E0C4-FA8D-4704-96C0-5BB1F9A5E4BD](https://user-images.githubusercontent.com/35382971/108751114-d5641f80-750f-11eb-88d2-6a20f24ff4a3.png)

it still amazes me how after all this time we've spent trying to improve the installation docs, the skill of basic reading comprehension still eludes many skids

at this point we'll have to resort to grunting and pointing at shapes and colors

**Comment by ArcerionDev** [comment]
remember that most of our fanbase are 1st grade skids with too much internet access

**Reply by ghost** [reply]
when i share it they're not sure how to scroll to the bottom of the page like they dont see the scroll bar or it doesnt go into their mind that "oh look i dont know what to do but ive only read half the page let me read the other half" so yes we need to resort to big shapes and colors that point them to "WIKI" tab

also i feel like whenever someone asks how to hack we need to stop linking them to this repository and instead link them to a video tutorial because that works better

**Comment by ArcerionDev** [comment]
@PatheticMustan please lock or delete that gist

**Comment by irhrhd** [comment]
That is probably annoying i feel your pain

**Comment by CyberXVI** [comment]
People who joined the discord i hope you read the rules
I sure did

**Comment by ghost** [comment]
How come people still not now the instructions? You’ve explained them so clearly, my mind hurts Natsumi.
